### PR TITLE
DVX-59 python: added flake8 & unit tests & renamed into python-listener

### PR DESCRIPTION
**Pull request by jean-noumena**

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[flake8]
+extend-ignore = E203
+exclude =
+    .git,
+    __pycache__,
+    generated,
+max-complexity = 10
+max-line-length = 119
```
Comment (ari-noumena) [inline .flake8:8]: 119 is a very non-specific number 😄 -- why not 120?

File: python-listener/test/unit_test.py
Comment (ari-noumena) [inline python-listener/test/unit_test.py:1]: Nit: maybe not the best name 😆

File: python-listener/Dockerfile [hunk 2]
```diff
@@ -14,4 +14,4 @@ RUN pip install --no-cache-dir -r requirements.txt
 EXPOSE 80
 
 # Run main.py when the container launches
```
Comment (pierre-noumena) [inline python-listener/Dockerfile:16]: Adjust comment to new python filename

Comment (pierre-noumena) [reply python-listener/Dockerfile:16]: Also need to adjust paas.mk to reflect new directory (python-listener) and new python file to run (app.py)

**Review by pierre-noumena — APPROVED** [review]

**Pull request closed without merging**